ext/node/update_node_stream.js

```diff
@@ -0,0 +1,507 @@
+#!/usr/bin/env -S deno run --allow-read --allow-write --allow-env --allow-run
+// deno-lint-ignore-file
+// Copyright 2018-2025 the Deno authors. MIT license.
+
+// This file is used to transform Node.js internal streams code to
+// Deno polyfills.
+
+import jscodeshift from "npm:jscodeshift@0.15.2";
+import $ from "jsr:@david/dax@0.42.0";
+
+import path from "node:path";
+
+import { version } from "../../tests/node_compat/runner/suite/node_version.ts";
+import { expandGlobSync } from "jsr:@std/fs@1.0.14/expand-glob";
+
+const globs = [
+  "internal/streams/*.js",
+  "stream/*.js",
+];
+
+// These have special handling for lazy loading
+const ignore = ["duplexify.js"];
+
+const moduleMap = {
+  "events": "node:events",
+  "buffer": "node:buffer",
+  "stream": "node:stream",
+  "string_decoder": "node:string_decoder",
+  "internal/abort_controller": "ext:deno_web/03_abort_signal.js",
+  "internal/events/abort_listener":
+    "ext:deno_node/internal/events/abort_listener.mjs",
+  "internal/assert": "ext:deno_node/internal/assert.mjs",
+  "internal/webstreams/adapters":
+    "ext:deno_node/internal/webstreams/adapters.js",
+  "internal/webstreams/compression": "ext:deno_web/14_compression.js",
+  "internal/webstreams/encoding": "ext:deno_web/08_text_encoding.js",
+  "internal/errors": "ext:deno_node/internal/errors.ts",
+  "internal/event_target": "ext:deno_node/internal/event_target.mjs",
+  "internal/util": "ext:deno_node/internal/util.mjs",
+  "internal/util/debuglog": "ext:deno_node/internal/util/debuglog.ts",
+  "internal/validators": "ext:deno_node/internal/validators.mjs",
+  "internal/encoding": "ext:deno_web/08_text_encoding.js",
+  "internal/blob": "ext:deno_web/09_file.js",
+};
+
+// Use default export for these conditional require()
+const defaultLazy = [
+  "internal/streams/passthrough",
+  "internal/streams/readable",
+  "internal/streams/duplexify",
+];
+
+// Workaround a bug in our formatter: "export default from;" does not work
+// correctly, so we rename it to something else and export.
+const renameForDefaultExport = ["from"];
```

Comment:
can you open a dprint issue and link it here?

Reply:
https://github.com/dprint/dprint-plugin-typescript/issues/705

Reply:
It looks like an swc syntax error.

Reply:
it can parse it on the swc [playground](https://play.swc.rs/?version=1.11.20&code=H4sIAAAAAAAAA0vOzysuUUgrys9VsFWorrXmSq0oyC8qUUhJTUsszYHIWAMAg8l5qiUAAAA%3D&config=H4sIAAAAAAAAA1WPSw7DIAwF9zkF8rrbdtE79BCIOhERP9mOVBTl7oUE0maH3xszsA5KwcwGnmotxzIkTYx0ziXhHER%2FSgKSE7IhmwRuvRWu1agd4x5tRwOiaUKpW8j3hoOLkbHjLfM22DH%2FC030iZD5ClZUh8nhVTc0Jfj4XvayfaQ%2B9tA%2F4Ad12XkxWH71TaEFh%2B0LYuVI0xQBAAA%3D)

Reply:
It's an swc diagnostic that's not surfaced in the playground. I'll open an swc issue.

Reply:
Looked into it and here's an example of the swc playground erroring: https://play.swc.rs/?version=1.11.20&code=H4sIAAAAAAAAA0vOzysuUUgrys9VsFWorrXmSq0oyC8qUUhJTUsszYHIWAMAg8l5qiUAAAA%3D&config=H4sIAAAAAAAAA1WPsQ6DMAxEd74Cee7aDp2rbv0IKzUoKCGRbSQQ4t8JkNCyxXfPd%2FFc1TV0YuBZz%2BmZhogsxOecFJl6xTEpQMajGLZR4VZcGmNgfVGDg9M3B59A5YFOoJNtt0EntEvL4YAit6R7rNxzHrgQhAqeNW9720z%2FPzLBRyaRK7ih2LeOrnVVrgQfvsNu5kt1inTUP%2BAHlbIzGKx8yuZ2WLWst%2B%2Fb9zUBAAA%3D

It's because of the `export default from` proposal.

Reply:
https://github.com/swc-project/swc/issues/10372